app/views/layouts/application.html.erb

```diff
@@ -7,19 +7,35 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <div class="container">
-    <div class="page-header">
-      <h1>Welcome to Tutorify</h1>
-    </div>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
```

Comment:
They create the side button on the nav bar when you have it adjust for mobile. See image below
![screen shot 2016-03-03 at 5 57 19 pm](https://cloud.githubusercontent.com/assets/11605201/13512565/72d360a4-e169-11e5-8a3f-777f080d66fd.png)